examples/testresample.rs

```diff
@@ -20,6 +20,9 @@ fn main() {
     st.set_rate(RATE, rate);
     st.skip_zeros();
 
+    st.set_quality(10);
+    println!("Quality: {}", st.get_quality());
```

Comment:
This could go to `stderr`.

Reply:
this is what allows the tests to pass - see pull request #1  :-)

perhaps we need to rethink that

Reply:
Yes. As I said the task is about taking the C code from speexdsp and convert it to rust to confirm that the wrapper is behaving correctly.

You might notice that the speexdsp-rs example and the speexdsp-sys example have the same output.

Reply:
ok, so if you need a visual comparison (C and Rust) of the output of the resampling, then we should leave it like you said.

... and write specific tests to f.e. check the set/get quality API ([like you suggest here](https://github.com/lu-zero)) instead of polluting this example. how about that?

Reply:
Sure, you could just add the same print on the [sys example](https://github.com/rust-av/speexdsp-rs/blob/master/speexdsp-sys/examples/testresample.rs) if you want :)